app/src/main/res/values/styles.xml

```diff
@@ -62,9 +62,20 @@
         <item name="colorAccent">@color/accent</item>
     </style>
     
-    <!--Theme for Spash-->
+    <!--Theme for Splash-->
     <style name="SplashTheme" parent="Theme.AppCompat.NoActionBar">
         <item name="android:windowBackground">@drawable/splash_background</item>
     </style>
 
+    <!--Redesign Login Screen-->
+    <style name="TextInputLayoutLogin" parent="Widget.MaterialComponents.TextInputLayout.OutlinedBox.Dense">
```

Comment:
We can reuse this theme in other places, ryt? If yes, then let's rename it to something generic.

Reply:
Yes, we can use it in other places as well. How about "TextInputLayout" or "TextInputLayout_StrokeWhite" ? If you have anything else in mind do let me know.